config/config.py

```diff
@@ -1,7 +1,8 @@
-from datetime import datetime
+from datetime import datetime, timezone
 from os import getcwd
 
 timestamp = datetime.today()
+timestamp_utc = datetime.now(timezone.utc).replace(tzinfo=timezone.utc)
```

Comment:
Why we need two timestamps in different timezones? I think we should always use the `utc` one.

Reply:
The first one is used for the isoformat.